core/domain/summary_services_test.py

```diff
@@ -196,7 +196,8 @@ def test_get_displayable_exp_summary_dicts_matching_ids(self):
             'status': 'public',
             'tags': [],
             'thumbnail_bg_color': '#a33f40',
-            'thumbnail_icon_url': '/images/subjects/Lightbulb.svg',
+            'thumbnail_icon_url': self.get_static_asset_url(
+                '/assets/images/subjects/Lightbulb.svg'),
```

Comment:
If all static asset URLs start with /assets, do we actually need to pass /assets in? I was thinking that perhaps this could be handled internally in get_static_asset_url().